vpr/src/draw/draw.cpp

```diff
@@ -694,21 +743,23 @@ void toggle_routing_bounding_box(GtkWidget* /*widget*/, gint /*response_id*/, gp
         draw_state->show_routing_bb = new_value;
 
     //redraw
-    if ((int)(draw_state->show_routing_bb) == (int)((int)(route_ctx.route_bb.size()) - 1)) {
+    if ((int)(draw_state->show_routing_bb)
```

Comment:
There are so many formatting edits that split a line into multiple lines. You might be using an IDE and setting the text wrap option to a small values which causes all these edits.

Reply:
@PaulaPerdomo : you should get rid of the short lines in that case, and rely on
make format
to adjust the final format.